### DISCUSSION TITLE
Base Entitiy의 컬럼들을 조회하지 않을 방법이 있을까요?

### DISCUSSION THREAD

**Discussion by YeaChan05** [KakaoFunding/back-end#22] [Q&A]
저희 엔티티들에 보면 다음과같이 시간 정보를 위한 구현 클래스들이 종종 있습니다
```Java
public class Category extends BaseTimeEntity {
```
문제는 이와 같은 엔티티들을 조회할 때마다 불필요한 쿼리들이 발생하게 됩니다

예를들어...

```SQL
select
        c1_0.category_id, 
        c1_0.created_at, -- 조회가 필요하지 않는 컬럼까지 조회됨
        c1_0.name,
        c1_0.parent_id,
        c1_0.updated_at -- 이것두...
    from
        category c1_0 
    where
        c1_0.category_id=?
```
시간정보가 필요하지 않는 조회 로직에 자꾸 생성 시간과 수정 시간이 끼어들게됩니다
그렇다고 매번 projection하기에는 번거롭기도 하고...
그래서 특별히 해결방법이 있는지 여쭤보고싶습니다!
@kmw2378 @hwan2-99

**Comment by kmw2378** [comment]
엔티티 상속은 어쩔 수 없는거 같아요ㅜ `Projections`를 사용해 DTO 조회를 하는게 제일 좋을 것 같습니다!
